react/new/src/constants/nhlColors.ts

```diff
@@ -0,0 +1,36 @@
+import { TeamColors } from '../types/theme'
+
+export const NHL_TEAM_COLORS: Record<string, TeamColors> = {
```

Comment:
Future idea: We should consider adding countries here too (thinking mostly the Swedes and Finns would appreciate that)

Reply:
Sky's the limit for sure! No objections